pkg/sarif.go

```diff
@@ -626,6 +626,13 @@ func (r *ScorecardResult) AsSARIF(showDetails bool, logLevel log.Level,
 
 	for _, check := range r.Checks {
 		check := check
+
+		// If check is exempted, skip
+		exempted, _ := check.IsExempted(r.Config)
+		if exempted {
+			continue
+		}
```

Comment:
nit: the code isn't that complicated/long, so the comment doesnt add much as written.

The comment would be better if you say why we're skipping. In this case something that mentions SARIF is used for code scanning, so we dont want alerts for things the owner knows about?

What do you think?

Reply:
Makes a lot of sense. Sorry about that. Will fix.